projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -4579,11 +4582,12 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
     public deleteRowById(rowId: any): any {
         const args = {
             rowID: rowId,
-            primaryKey: rowId,
-            cancel: false,
+            key: rowId,
```

Comment:
if `args` is typed as `IRowDataCancelableEventArgs`, then `rowID` is marked as mistake:

![image](https://github.com/IgniteUI/igniteui-angular/assets/14248932/414f0e9e-bf10-4661-8b44-63692fa52ac6)

Is `IRowDataCancelableEventArgs` expedted to omit `rowID` from the interface?

Reply:
And shouldn't we primaryKey: rowId, in order to be consistent when IRowDataEventArgs is emitted from rowDelete and rowDeleted 
![image](https://github.com/IgniteUI/igniteui-angular/assets/46093564/f3fe0542-6942-4f9c-8e65-da7059dfa65c)